usecases/modules/vectorizer_test.go

```diff
@@ -208,26 +211,6 @@ func TestProvider_UpdateVector(t *testing.T) {
 		assert.Nil(t, err)
 	})
 
-	t.Run("with nonexistent class", func(t *testing.T) {
```

Comment:
why this test had to be removed?

Reply:
It was testing a weird thing: "what happens when you pass an object with a different class name than the passed class", however, it was asserting for missing moduleconfig.

I've added better tests for this.